README.md

```diff
@@ -100,6 +103,6 @@ I would like to thank to the code reviewers and my partners who are helping all
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed. 
+This project is [MIT](https://github.com/Gleennkar/blog_app/LICENSE) licensed. 
 
```

Comment:
- [ ] As mentioned by the previous reviewer, I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](https://github.com/Gleennkar/blog_app/LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**